internal/usecase/interactor/usecase.go

```diff
@@ -0,0 +1,121 @@
+package interactor
+
+import (
+	"context"
+
+	"github.com/reearth/reearth-backend/internal/usecase"
+	"github.com/reearth/reearth-backend/internal/usecase/interfaces"
+	"github.com/reearth/reearth-backend/internal/usecase/repo"
+	"github.com/reearth/reearth-backend/pkg/id"
+)
+
+type uc struct {
+	tx             bool
+	readableTeams  id.TeamIDList
+	writableTeams  id.TeamIDList
+	readableScenes id.SceneIDList
+	writableScenes id.SceneIDList
+}
+
+func Usecase() *uc {
+	return &uc{}
+}
+
+func (u *uc) CanReadTeams(ids ...id.TeamID) *uc {
```

Comment:
@rot1024 
`Can` in the method name gives an impression that this method will return a bool value!
I think using a name like `WithReadableTeams` would be better